common/src/main/java/dev/schmarrn/lighty/api/LightyHelper.java

```diff
@@ -29,21 +29,31 @@ private static boolean specialCases(Block block) {
         return block instanceof CarpetBlock;
     }
 
-    public static float getOffset(BlockState block, BlockPos pos, ClientLevel world) {
-        float offset = 0;
-        if (block.getBlock() instanceof SnowLayerBlock) { // snow layers
-            int layer = world.getBlockState(pos).getValue(SnowLayerBlock.LAYERS);
+    public static float getOffset(BlockState blockState, BlockState blockStateUp) {
```

Comment:
it would be great if this method only took one blockstate and returns the correct offset of that block and then work with just that

Reply:
Is there a way to get a POS from a Block or BlockState? Otherwise I'll revert the signature of the function.

Reply:
> it would be great if this method only took one blockstate and returns the correct offset of that block and then work with just that

Is there a way to get a POS from a Block or BlockState? Otherwise I'll revert the signature of the function.